spec/tools/miqldap_to_sssd/miqldap_configuration_spec.rb

```diff
@@ -0,0 +1,28 @@
+$LOAD_PATH << Rails.root.join("tools", "miqldap_to_sssd").to_s
+
+require "miqldap_configuration"
+
+describe MiqLdapToSssd::MiqLdapConfiguration do
+  before do
```

Comment:
Why are we stubbing the logger?

Reply:
Thank you @bdunne 

Without stubbing LOGGER the following error is produced: `uninitialized constant LOGGER`

Can you suggest a way to avoid stubbing the logger or is this acceptable?

Reply:
You could require `MiqLdapToSssd` which defines it.

Reply:
I think that's a good idea, but let's do that in a separate PR as the LOGGER stub occurs in other files too.